src/_pytest/faulthandler.py

```diff
@@ -0,0 +1,102 @@
+import io
+import os
+import sys
+
+import pytest
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup("terminal reporting")
+    group.addoption(
+        "--no-faulthandler",
+        action="store_false",
+        dest="fault_handler",
+        default=True,
+        help="Disable faulthandler module.",
+    )
+
+    group.addoption(
+        "--faulthandler-timeout",
+        type=int,
```

Comment:
This should probably be `float`, to allow fractional second timeouts.

Reply:
Thanks, done!